tests/python/common/data/test_data.py

```diff
@@ -775,54 +776,55 @@ def assert_data(lhs, rhs, size, node=False):
     assert expect_except
 
 
-def _test_DefaultDataParser():
+def _get_data_table(data_frame):
     from dgl.data.csv_dataset_base import DefaultDataParser
 
-    # common csv
     with tempfile.TemporaryDirectory() as test_dir:
         csv_path = os.path.join(test_dir, "nodes.csv")
-        num_nodes = 5
-        num_labels = 3
-        num_dims = 2
-        node_id = np.arange(num_nodes)
-        label = np.random.randint(num_labels, size=num_nodes)
-        feat = np.random.rand(num_nodes, num_dims)
-        df = pd.DataFrame(
-            {
-                "node_id": node_id,
-                "label": label,
-                "feat": [line.tolist() for line in feat],
-            }
-        )
-        df.to_csv(csv_path, index=False)
+
+        data_frame.to_csv(csv_path, index=False)
         dp = DefaultDataParser()
         df = pd.read_csv(csv_path)
-        dt = dp(df)
-        assert np.array_equal(node_id, dt["node_id"])
-        assert np.array_equal(label, dt["label"])
-        assert np.array_equal(feat, dt["feat"])
+
+    with warnings.catch_warnings():
```

Comment:
I am not a big fan of catch warning and hide, it is no big deal to output warnings in the log.

Reply:
@frozenbugs: I think that in general, you are right - it's okay to display warnings in the log. On the other hand, with a large number of warnings, it can be very difficult to find any new ones that can be very important. 

The warning we saw here ("Unamed column found. Ignored...") is one of those that could create this kind of white noise. 

By the way, `Unamed` (and not `Unnamed` as it should be) is used twice in
- `data/csv_dataset_base.py`
- `data/data-loadcsv.rst`.

Do you want me to correct this typo?

Reply:
Yes, if there is a typo, it is always nice to have a quick fix.

For "Unamed column found" warning, what about just give it a random name in the test?